scilpy/image/datasets.py

```diff
@@ -84,7 +85,7 @@ def is_voxel_in_bound(self, i, j, k):
         return (0 <= i < self.dim[0] and 0 <= j < self.dim[1] and
                 0 <= k < self.dim[2])
 
-    def get_voxel_at_position(self, x, y, z):
+    def voxmm_to_idx(self, x, y, z, origin='center'):
```

Comment:
Is the new `origin` argument useful? Why would we need this kind of flexibility? Could we not just set it to one or the other and take it into account when writing our tracking scripts?

Reply:
That dataset is not necessarily only associated with our traking script, though. It could eventually be used elsewhere, so I tought it was better to be thorough, like in the sft. But I won't object if others agree.

Reply:
Yes ok. But in this case we need to be extra careful when writing our tracking scripts, because the `SeedGenerator` always returns positions with origin `corner`. Should the `SeedGenerator` have an origin argument too then?

Reply:
I added one. For now it is not used-defined (self.origin='corner') but at least a bit more obvious?